templates/arnold/render-linux-frames/job.template.json

```diff
@@ -53,7 +53,7 @@
                 "poolId": "[parameters('poolId')]"
             },
             "jobPreparationTask": {
-                "commandLine": "env;export"
```

Comment:
did export get removed on purpose?

Reply:
I'll add it back in like so 
                "commandLine": "/bin/bash -c 'env;export'"